projects/packages/videopress/src/client/block-editor/blocks/videopress/components/videopress-uploader/index.js

```diff
@@ -194,6 +198,34 @@ const VideoPressUploader = ( { attributes, setAttributes, noticeUI, noticeOperat
 		tusUploader.current = videoPressUploader( file );
 	};
 
+	const startUploadFromLibrary = attachmentId => {
+		const path = `videopress/v1/upload/${ attachmentId }`;
+		apiFetch( { path, method: 'POST' } ).then( result => {
```

Comment:
Question: In case of a network failure or something like this, does `apiFetch` catch the error internally? Asking because there is no `.catch`

Reply:
Nevermind, just saw the `.catch` in `onSelectVideo`.

Reply:
Actually, you might be right. We need a `catch` there

Reply:
https://github.com/Automattic/jetpack/pull/25792/commits/1dc774f61426c27bfc5f70cb0d190360419c3a72